src/h3lib/lib/linkedGeo.c

```diff
@@ -294,7 +294,7 @@ int normalizeMultiPolygon(LinkedGeoPolygon *root) {
     // We assume that the input is a single polygon with loops;
     // if it has multiple polygons, don't touch it
     if (root->next) {
-        return NORMALIZATION_ERR_MULTIPLE_POLYGONS;
+        return NORMALIZATION_ERR_MULTIPLE_POLYGONS; // Do we want to add these error codes to the public API?
```

Comment:
I think we should, though I think at present the user will never encounter these. The current input for `normalizeMultiPolygon` comes from `h3SetToVertexGraph`, so I don't think these error codes are reachable through the public API unless there's an error within the library. I'm not sure what to do with these internal errors, but I think it makes sense to unify them with external errors in order to a) have one approach to error handling, and b) not require a refactor if this ever is exposed in a way that could be prone to user input errors. 

That said, we might want to note in the docs that these errors are internal fail-safes, and if you hit them you should file a bug.

Reply:
Makes sense.

Reply:
Moved discussion to https://github.com/uber/h3/issues/628